.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "go.coverOnSingleTest": true,
+    "go.coverOnSingleTestFile": true,
+    "go.testFlags": ["-v", "-race"]
+}
```

Comment:
```suggestion
}

```

Reply:
I may or may not have added this. It seems different on my current machine (Windows is new to me) and the github view.